homeassistant/components/vacuum/__init__.py

```diff
@@ -96,6 +96,7 @@ class VacuumEntityFeature(IntFlag):
 
 
 # These SUPPORT_* constants are deprecated as of Home Assistant 2022.5.
+# Will be removed with VacuumEntity in Home Assistant Core 2024.2
```

Comment:
@frenck Do we require an additional developer blog entry or is the [deprecation dev blog entry](https://developers.home-assistant.io/blog/2022/04/02/support-constants-deprecation/?_highlight=vacuumentityfeature) enough?